Bottom: 27db1321fde89455afc87de24c9a9f468b59b08c
Top:    2347bd7497f4b54721e2fdcb0cd00754b30b7659
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-30 14:18:57 +0800

Refresh of ptr_ring-introduce-batch

---

diff --git a/include/linux/ptr_ring.h b/include/linux/ptr_ring.h
index 4771ded..ec3369d 100644
--- a/include/linux/ptr_ring.h
+++ b/include/linux/ptr_ring.h
@@ -251,13 +251,13 @@ static inline int __ptr_ring_consume_batched(struct ptr_ring *r,
 					     void **array, int n)
 {
 	void *ptr;
-	int i = 0;
+	int i;
 
-	while (i < n) {
+        for (i = 0; i < n; i++) {
 		ptr = __ptr_ring_consume(r);
 		if (!ptr)
 			break;
-		array[i++] = ptr;
+		array[i] = ptr;
 	}
 
 	return i;
